src/plugins/taxonomy/components/hashtags-picker-field/index.jsx

```diff
@@ -78,6 +85,7 @@ const HashtagsPickerField = ({
         className="hidden-options-toggle"
         closeOnSelect={false}
         creatable
+        styles={customStyles}
```

Comment:
@nanetteCodes just put the object straight in here, dont need the customStyles var